jest.config.ts

```diff
@@ -6,6 +6,11 @@ module.exports = {
     '^@/(.*)$': '<rootDir>/src/$1',
     '\\.(css|less)$': '<rootDir>/__mocks__/styleMock.ts',
     '\\.(gif|ttf|eot|svg)$': '<rootDir>/__mocks__/fileMock.ts',
+    '^styled-theming$': '<rootDir>/__mocks__/styledThemingMock.ts',
+    '^mdi-react/EyeIcon$': '<rootDir>/__mocks__/EyeIconMock.tsx',
```

Comment:
if some icon is only used on one page, it shouldn't be introduced globally. I suggest mock it in each test file.

Reply:
Removed mocks from global